pkg/apps/controller/deployer/factory.go

```diff
@@ -124,6 +126,15 @@ func (c *DeploymentController) updateReplicationController(old, cur interface{})
 	c.enqueueReplicationController(curRC)
 }
 
```

Comment:
Add a comment on why do we need to resync the RC when the deployer is observed in the cache.